src/ess/wfm/beamline.py

```diff
@@ -0,0 +1,125 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+import scipp as sc
+import numpy as np
+from .choppers import Chopper, make_chopper_cascade
+
+# class Beamline:
+#     def __init__(self, choppers, source):
+#         self.choppers = choppers
+#         self.source = source
+
+
+def make_fake_beamline(
+        chopper_positions={
+            "WFMC1": sc.vector(value=[0.0, 0.0, 6.775], unit='m'),
+            "WFMC2": sc.vector(value=[0.0, 0.0, 7.225], unit='m')
+        },
+        frequency=sc.scalar(56.0, unit=sc.units.one / sc.units.s),
+        lambda_min=sc.scalar(1.0, unit='angstrom'),
+        pulse_length=sc.scalar(2.86e-03, unit='s'),
+        pulse_t_0=sc.scalar(1.3e-4, unit='s'),
+        nframes=2):
+    """
+    Fake chopper cascade with 2 optically blind WFM choppers.
+    Based on mathematical description in Schmakat et al. (2020);
+    https://www.sciencedirect.com/science/article/pii/S0168900220308640
+    """
+
+    dim = 'frame'
```

Comment:
```suggestion
    dim_frame = 'frame'
```
Helps with readability further down

Reply:
I don't see any other `dim` in that function or file, so i left it as is.